### PR TITLE
dev-libs/ocl-icd: bump to 2.2.12, fix multilib

### PR DESCRIPTION
**Pull request by gyakovlev**
Also switch to ruby-single eclass
Bug: https://bugs.gentoo.org/637964
Package-Manager: Portage-2.3.41, Repoman-2.3.9

### CODE REVIEW

**Comment by gyakovlev** [comment]
@patricklauer can you please take a look?

File: dev-libs/ocl-icd/ocl-icd-2.2.12.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	local OCL_DIR="${D%/}/usr/$(get_libdir)/OpenCL/vendors/ocl-icd"
+	mkdir -p "${OCL_DIR}" || die "mkdir failed"
+	mv "${D%/}/usr/$(get_libdir)"/libOpenCL* "${OCL_DIR}" || die
+
+	insinto /etc/OpenCL/vendors/
+	doins "${T}/ocl-icd-${ABI}.icd"
+}
+
+multilib_src_install_all() {
+	find "${ED%/}" -name '*.la' -delete
```
Comment (hydrapolic) [inline dev-libs/ocl-icd/ocl-icd-2.2.12.ebuild]: Missing die.

File: dev-libs/ocl-icd/ocl-icd-2.2.12.ebuild [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+inherit ruby-single multilib-minimal
+
+DESCRIPTION="Alternative to vendor specific OpenCL ICD loaders"
+HOMEPAGE="https://forge.imag.fr/projects/ocl-icd/"
+SRC_URI="https://forge.imag.fr/frs/download.php/836/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```
Comment (hydrapolic) [inline dev-libs/ocl-icd/ocl-icd-2.2.12.ebuild]: Empty variable can be omitted.

File: dev-libs/ocl-icd/ocl-icd-2.2.12.ebuild (hunk 1 above)
Comment (gyakovlev) [reply dev-libs/ocl-icd/ocl-icd-2.2.12.ebuild]: thanks! totally missed that.

File: dev-libs/ocl-icd/ocl-icd-2.2.12.ebuild (hunk 2 above)
Comment (gyakovlev) [reply dev-libs/ocl-icd/ocl-icd-2.2.12.ebuild]: I usually keep empty IUSE. no reason behind it)

**Pull request closed without merging**